src/templates/finops-hub/modules/dataFactory.bicep

```
@@ -381,7 +426,7 @@ resource linkedService_storageAccount 'Microsoft.DataFactory/factories/linkedser
 }
 
 resource linkedService_dataExplorer 'Microsoft.DataFactory/factories/linkedservices@2018-06-01' = if (deployDataExplorer) {
-  name: 'hubDataExplorer'
+  name: dataExplorerName
```

Is there a reason you want the name to change for every deployment? Having a static name for every hub instance will streamline troubleshooting and supportability. I'm actually thinking we should do this with storage as well.
```suggestion
  name: 'hubDataExplorer'
```

---

Done